src/test/util_tests.cpp

```diff
@@ -2456,4 +2456,49 @@ BOOST_AUTO_TEST_CASE(remove_prefix)
     BOOST_CHECK_EQUAL(RemovePrefix("", ""), "");
 }
 
+BOOST_AUTO_TEST_CASE(util_ParseByteUnits)
```

Comment:
Maybe add a test for unknown suffix, e.g. `123X`.

Reply:
Added invalid unit test